merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/GetPlanAction.java

```diff
@@ -10,39 +10,17 @@
 import java.sql.PreparedStatement;
 import java.sql.SQLException;
 import java.util.Map;
+import java.util.Optional;
 
 /*package-local*/ final class GetPlanAction implements AutoCloseable {
   private static final @Language("SQL") String sql = """
-    with
```

Comment:
I'm curious about the tradeoff of making actions one-to-one with tables. One one hand, it seems like a very clean separation of database implementation details from model types. On the other hand, it seems like there are situations where we would now have to call multiple actions (and thus take multiple trips to the database) where before we could use SQL to join multiple tables together.

Reply:
Though most actions do only interact with one table, this is not the case for all. Examine the `LookupSimulationDatasetAction` class, which reads from 5 tables in a single query! That said, I do believe I've seen some areas where we perform multiple operations where we could have performed one. Much of this is an artifact of my inexperience when I was first working on Postgres actions. My rope bridge, if you will.

Reply:
Exactly. The actions provide a record of the state of the database that's useful to the `PostgresResultsCellRepository`.